andela/AndelaEatsUI
# #162050694 Create tab navigation for vendors

#### What does this PR do?
Tab Navigation for Vendors

#### Description of Task to be completed?
Create reusable Tab component  for Vendors 

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git) and
-  cd into AndelaEatsUI and checkout to this branch `ft-create-reusable-tab-component-162050694`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, click on  `Vendors`
- run `npm test` to ensure all tests are passing

#### What are the relevant pivotal tracker stories?
- [#162050694](https://www.pivotaltracker.com/n/projects/2178749/stories/162050694)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1119" alt="screen shot 2018-11-19 at 2 45 04 pm" src="https://user-images.githubusercontent.com/21138774/48710786-bdebad00-ec09-11e8-8946-678a61081b10.png">
